components/ui-shell/index.ts

```diff
@@ -18,6 +18,11 @@ import styles from './index.scss';
 import { Variant, NAV_ITEMS, SOCIAL_LINKS } from './settings.js';
 import type { NavItem, TopLevelNavItem } from './settings.js';
 
+const BASE_URL =
```

Comment:
What do you think to change it to something like `LEARN_BASE_URL`?

Reply:
I like it @abdonrd

Reply:
Nice! Then we can merge it and release a new version!

Reply:
Let me know if this works

Addressed w/ https://github.com/Qiskit/web-components/pull/183/commits/a0c0fe984f409d97c88233be715137667000d183